vue-frontend/src/portfolio-json/justly-data.js

```diff
@@ -35,6 +35,9 @@ import branding4Image800w from "@/assets/images/portfolio/justly/branding/justly
 import branding4Image1200w from "@/assets/images/portfolio/justly/branding/justly_branding_4-1200w.webp";
 import branding4Image1600w from "@/assets/images/portfolio/justly/branding/justly_branding_4-1600w.webp";
 
+import playStoreImage from "@/assets/images/portfolio/play_store.webp";
```

Comment:
If images are same, let's use them directly in the components instead of init it in all the data